TypeTreeDumper/ExportOptions.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace TypeTreeDumper
+{
+    public class ExportOptions
+    {
+        public string OutputDirectory { get; internal set; }
```

Comment:
This should be `{ get; init; }` rather than `{ get; internal set; }`

Reply:
`init` is not available on your net version

Reply:
Ah right, since this project is still on .NET Framework. I think a `public` setter is fine for now then, until the move to .NET 6 or later is made.